CMakeLists.txt

```diff
@@ -141,6 +141,13 @@ list(APPEND OLIVE_LIBRARIES
 
 # Link PortAudio
 find_package(PortAudio REQUIRED)
+set(CMAKE_REQUIRED_INCLUDES ${PORTAUDIO_INCLUDE_DIRS})
+include(CheckIncludeFileCXX)
+check_include_file_cxx( "pa_jack.h" PA_HAS_JACK)
+if (PA_HAS_JACK)
+  add_compile_definitions(PA_HAS_JACK)
```

Comment:
`list(APPEND OLIVE_DEFINITIONS -DPA_HAS_JACK)` would be preferable to the global `add_compile_definitions`

Reply:
True, I didnt notice due tunnel vision from intially trying to get it to work in app/audio/CMakeLists.txt instead of doing it in the root CMakeLists.txt. Ill change this and then rebase.